components/Cms/Element/CmsElementImageSlider.vue

```diff
@@ -1,53 +1,85 @@
 <script setup lang="ts">
+import LayoutSlider from '~/components/Layout/Slider/LayoutSlider.vue';
+import LayoutSliderSlide from '~/components/Layout/Slider/LayoutSliderSlide.vue';
+
 const props = defineProps<{
     element: CmsElementImageSlider;
 }>();
+// Main slider html object
+const sliderRef = ref(null);
+
+const config = computed(() => props.element.config);
+const sliderItems = computed(() => props.element.data?.sliderItems ?? []);
+const slides = ref(Array.from({ length: sliderItems.value.length }));
 
-const config = useCmsElementConfig(props.element);
-const sliderItems = config.getConfigValue('sliderItems');
+console.log(props.element.config);
+if (sliderItems.value.length > 0) {
+    // Main slider swiper instance
+    useSwiper(sliderRef, {
+
+    });
+}
+const autoplayConfig = computed(( )=>{
+    if (config.value?.autoSlide.value) {
+        return {
+            delay: config.value?.autoplayTimeout.value,
+            disableOnInteraction: false,
+        };
+    } else {
+        return false; // Wenn autoplay deaktiviert sein soll
+    }
+});
```

Comment:
```suggestion
const autoplayConfig = computed(( )=>{
    if (config.value?.autoSlide.value) {
        return {
            delay: config.value?.autoplayTimeout.value,
            disableOnInteraction: false,
        };
    }
    
     return false;
});
```

the else block is not needed since you return in the if statement